dashboard/config/routes.rb

```diff
@@ -369,6 +369,7 @@ module OPS
       post :facilitator_program_registrations, to: 'facilitator_program_registrations#create'
       post :regional_partner_program_registrations, to: 'regional_partner_program_registrations#create'
 
+      post :pre_workshop_surveys, to: 'pre_workshop_surveys#create'
```

Comment:
Should this be pre_workshop_survey (singular) instead?

Reply:
For this I followed the existing pattern (see next 2 lines)